group-coordinator/src/main/java/org/apache/kafka/coordinator/group/modern/TopicIds.java

```diff
@@ -263,6 +263,7 @@ public boolean remove(Object o) {
         throw new UnsupportedOperationException();
     }
 
+    @SuppressWarnings("rawtypes")
```

Comment:
Not clear to me, why updating parameter `Collection` did not work... Tried `Collection<?>` and `Collection<Uuid>` but compiler did not like it...

Reply:
```suggestion
    @Override
    public boolean addAll(Collection<? extends Uuid> c) {
        throw new UnsupportedOperationException();
    }
```

Reply:
I thought I tried this, too, but did not work... Let me double check. Maybe I did mess up.